tensorflow/python/keras/utils/generic_utils.py

```diff
@@ -211,6 +211,10 @@ def deserialize_keras_object(identifier,
         raise ValueError('Unknown ' + printable_module_name + ':' +
                          function_name)
     return fn
+  elif isinstance(identifier, list):
+    return [deserialize_keras_object(sub_identifier, module_objects,
```

Comment:
I think it would be preferable to have the list handling in `hdf5_format.py` above rather than here. `deserialize` is meant to handle single objects. The code duplication is minimal (one line).

Reply:
Sounds good, I'm on it.

Reply:
Done!